java/test/jmri/jmrix/can/cbus/simulator/CbusDummyNodeTest.java

```diff
@@ -55,6 +55,7 @@ public void testNodeRequestNumber() {
         Assert.assertTrue("start getSendIn", ((CbusSimCanListener) t.getCanListener()).getSendIn() );
         Assert.assertFalse("start getSendOut", ((CbusSimCanListener) t.getCanListener()).getSendOut() );
         Assert.assertTrue("start getsendsWRACKonNVSET", t.getsendsWRACKonNVSET());
+        Assert.assertFalse("start getsendsWRACKonNVSET", t.getnvWriteInLearnOnly());
```

Comment:
Maybe modify the error report (1st string) to refer to `getnvWriteInLearnOnly`?  Not expecting it to fail, so just a suggestion.